hardware/i2c.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+order:
+title: i2c
+heading: Schematics
+description: micro:bit i2c pins
+permalink: /hardware/schematic/
+ref: i2c
+lang: en
+assigned-to: markw
+review-with: jonnya
+---
+
+If you make an accessory for the micro:bit, please help us by editing the page and sharing the details of the i2c addresses you use.
+
+## Use of the shared I2C bus
+
+The motion sensors on the board are on the same I2C bus as the edge connector I2C pins. This means that if you have an accessory that uses I2C on this bus, you need to check it won’t clash with the new sensors.
+
+Because we want to avoid having to do another revision in the future, we are designing in two possible sensor devices, but all micro:bits will ship with only one combined magnetometer and accelerometer. The DAL will automatically select which driver to instantiate (and hence this will work in MicroPython and MakeCode without any issues).
```

Comment:
Either duplicate the language about FXOS here, or link to the appropriate section of the hardware page ('Motion Sensor')

Reply:
have duplicated FXOS language